app/ai/seasonal-advice/page.tsx

```
@@ -0,0 +1,478 @@
+                result.error ? (
+                  <div className="text-red-300 text-center py-4">
+                    <p className="font-semibold">Error:</p>
+                    <p>{result.error}</p>
+                  </div>
+                ) : (
+                  <div className="space-y-6">
+                    {typeof result === "string" ? (
+                      <div className="prose prose-invert max-w-none">
+                        <div
+                          className="text-green-200 leading-relaxed space-y-4"
+                          dangerouslySetInnerHTML={{
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)